packages/react/src/fonts.scss

```diff
@@ -0,0 +1,6 @@
+@import "@fontsource/roboto/100";
+@import "@fontsource/roboto/300";
+@import "@fontsource/roboto/400";
+@import "@fontsource/roboto/500";
+@import "@fontsource/roboto/700";
+@import "@fontsource/roboto/900";
```

Comment:
I don't know the rule for this repo, but you should probably add a new line at the end of this file.